projects/step-frontend/src/lib/modules/plan/components/pure-plan-editor/pure-plan-editor.component.html

```diff
@@ -0,0 +1 @@
+<step-plan-editor [id]="id"> </step-plan-editor>
```

Comment:
I dont like having the "pure plan editor" component so much. I would actually favour to directly use the plan-editor wherever we use the pure-plan-editor. Can you tell me why yout liked it better like this?

Reply:
I like your new naming sugestion, your general approach actually matches how we handle other compoennts so it makes sense to go with it.